templates/partials/contacts_content.html

```diff
@@ -125,7 +125,7 @@
                 </div>
               </div>
               <div ng-show="selected.doc.type === 'clinic'" mm-auth="can_view_tasks" class="col-xs-1 tasks-count">
-                <span class="badge">{{ child.doc.tasks.length || 0 }}</span>
+                <span class="badge">{{ (selected.tasks | filter:filterTasksByChild(child)).length || 0 }}</span>
```

Comment:
That's what I tried at first, but apparently the filter only goes one layer deep, unless you're willing to have the value match against every key with `$: foo`, which we don't want to do :-(